src/org/usfirst/frc4904/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.Idle;
+import org.usfirst.frc4904.standard.custom.sensors.CANTalonEncoder;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Arm extends Subsystem {
+	public static final double ELBOW_MULTIPLIER = 1.0;
+	public static final double WRIST_MULTIPLIER = 1.0;
+	public final Motor elbowMotorA;
+	public final Motor elbowMotorB;
+	public final Motor wristMotorA;
+	public final Motor wristMotorB;
+	public final CANTalonEncoder elbowEncoderA;
+	public final CANTalonEncoder elbowEncoderB;
+	public final CANTalonEncoder wristEncoderA;
+	public final CANTalonEncoder wristEncoderB;
+
+	public Arm(Motor elbowMotorA, Motor elbowMotorB, Motor wristMotorA, Motor wristMotorB,
```

Comment:
This is a large constructor. A couple of things —

1. Could you make a constructor that includes a name, so people can initialize "armthingo" or whatever?
2. More importantly, do these need to be cantalon encoders? could this be replaced with a broader class?